test/types/index.ts

```diff
@@ -111,6 +111,32 @@ server
     method: ['GET', 'POST', 'PUT'],
     url: '/route'
   })
+  .register(function (instance, options, done) {
```

Comment:
Could you make sure `done` called after `.get` inside this plugin? Without it the server does not start. I tested using `./node_modules/.bin/tsc test/types/index.ts --target es6 --module commonjs && node test/types/index.js`

Reply:
Sure, I'll update it.